prow/cmd/deck/pr_history.go

```diff
@@ -181,17 +181,36 @@ func updateCommitData(commits map[string]*commitData, org, repo, hash string, bu
 	}
 }
 
-func parsePullKey(key string) (org, repo string, pr int, err error) {
+// parsePullURL parses PR history URLs according to two possible formats:
```

Comment:
Can we just do one? Why honor both?

Reply:
I mainly want to keep the old way because it's a lot easier to remember and manually type out, and still works just fine for Github PRs.

Reply:
Updated to solely use the new format.